source/getting-started/welcome_email.rst

```diff
@@ -0,0 +1,55 @@
+Welcome Email to End Users
+===========================
+
+
+To make it easy for your end users to start using Mattermost right away, we created a sample email template that you can use.
+
+Remember to replace all the items below in bold with your information.
+
+
+Email Template
+-------------------
+
+From: **[company name]** IT Team
+
+To: End users
+
+Subject: New Communication Platform - Mattermost
+
+
+Hi all,
+ 
+As some of you already know, we are moving to Mattermost as our communication platform. Mattermost is a messaging app where you can talk, share files, and collaborate on projects or initiatives. Mattermost also integrates with many of the apps that you use every day, like **[add apps]**.  
+
+We are moving to Mattermost because it will allow us to become more productive by being able communicate in one place instead of using email, instant chat, and a multitude of other programs.
```

Comment:
Intro seems a bit wordy. Maybe it would be better to use our tag line. Something like,

"Mattermost will host all our communication in one place, is instantly searchable and available from all your devices."

Reply:
Updated